File: Majo/Act-Instructor_Henry/lenguajes/ruby/orientado_a_objeto/clase.rb
class Gato
    attr_accessor :edad, :nombre, :raza, :sexo, :color_ojos
  
    def initialize(raza, sexo, color_ojos)
      @raza = raza
      @sexo = sexo
      @color_ojos = color_ojos
    end
  end
  
  def main
    gato = Gato.new('Siames', 'Macho', 'Azul')
    puts gato.raza
    puts gato.sexo
    puts gato.color_ojos
    gato.nombre = "Garfield"
    gato.edad = 5
    puts gato.nombre
    puts gato.edad
  end
  
  main
  

File: Majo/Act-Instructor_Henry/lenguajes/dart/orientado_a_objeto/clase.dart
class Gato{
  int? edad;
  String? nombre;
  String? raza;
  String? sexo;
  String? color_ojos;

  Gato(String raza, String sexo, String color_ojos) {
    this.raza = raza;
    this.sexo = sexo;
    this.color_ojos = color_ojos;
  }
}

void main(){
  Gato gato = new Gato('Siames', 'Macho', 'Azul');
  print(gato.raza);
  print(gato.sexo);
  print(gato.color_ojos);
  gato.nombre = "Garfield";
  gato.edad = 5;
  print(gato.nombre);
  print(gato.edad);
}